From ea389814722933ce23de99a1ebadde6bb07f0e2c Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Mon, 13 Feb 2017 18:01:25 +0800
Subject: [PATCH 1511/5242] MLK-13917 pxp: fix build error for pxp library in
 user space

commit  436a9b9e8e91b5b53b72e77d24be735ec18ff5b4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix following build error by changing type to '__u64'.

include/uapi/linux/pxp_dma.h:230:2: error: unknown type name 'u64'
   u64 lut_sels;
   ^~~

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/pxp_dma.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/uapi/linux/pxp_dma.h b/include/uapi/linux/pxp_dma.h
index 948b9ee..d5d4c58 100644
--- a/include/uapi/linux/pxp_dma.h
+++ b/include/uapi/linux/pxp_dma.h
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2013-2015 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2017 NXP
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -227,7 +228,7 @@ struct pxp_proc_data {
 	bool combine_enable;
 
 	/* LUT cleanup */
-	u64 lut_sels;
+	__u64 lut_sels;
 
 	/* the mode pxp's working against */
 	enum pxp_working_mode working_mode;
-- 
1.7.9.5

